lib/tasks/version.rake

```diff
@@ -10,8 +10,12 @@ namespace :data do
         version.update_attributes(conference_id: version.item_id)
 
       elsif version.item_type == 'Event'
-        event = (version.item || version.reify || version.next.reify)
-
+        event = version.item
```

Comment:
how can happen that `version.item` is `null` but `version.item_id` references a valid event?

Reply:
If the item is let's say an Event, then the version record keeps version.item_id, let's say number 9 (as in 9 being the id of the Event record). Now that's a field on the database and the record won't get delete once Event.find(9) is destroyed.

`version.item` tries to give you the actual object Event.find(9), which may or may not be in our database. 

`version.reify` returns the item as it was one version before (if the version is the one where the event was destroyed, then it builds it as it was before its destruction)